### PR TITLE
VACMS-000 Reduce polling rate on site_alert to reduce load.

### PR DESCRIPTION
**Pull request by swirtSJW**
## Description

See _issueid_. 

## QA steps

As user _uid_ with _user_role_
1. Do this
1. Then that
1. Then validate Acceptance Criteria from issue
- [ ] This
- [ ] That
- [ ] The other thing

## Definition of Done
- [ ] Product release notes 
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: config/sync/block.block.sitealert.yml [hunk 1]
```diff
@@ -17,5 +17,5 @@ settings:
   label: 'Site Alert'
   provider: site_alert
   label_display: '0'
-  timeout: 20
+  timeout: 29
```
Comment (swirtSJW) [inline config/sync/block.block.sitealert.yml:20]: I bumped it to just under 30 seconds, which is our current deploy wait.   To go any higher, we would have to increase the deploy wait.

**Comment by ElijahLynn** [comment]
I'm not sure this is necessary, the PHP load on the servers was not high because of this polling, nor was the RDS load. I mainly want to see the Editor UX stay nice and if anything lowered the polling. I guess I'm okay with trying this out, but with the mindset that we revisit this after this deadlock issue is resolved.

**Comment by ElijahLynn** [comment]
How do you feel about closing this for now?

**Comment by swirtSJW** [comment]
This is no longer needed.  Closing

**Pull request closed without merging**